src/MSBuild.Sdk.SqlProj/Sdk/Sdk.targets

```diff
@@ -33,8 +33,6 @@
   </ItemGroup>
 
   <PropertyGroup>
-    <TargetExt>.dacpac</TargetExt>
-    <TargetPath>$(TargetDir)/$(MSBuildProjectName)$(TargetExt)</TargetPath>
```

Comment:
Just to clarify, this is the functional change? Once we remove the explicit `TargetPath`, MSBuild incorporates `TargetName` when it builds the path?

Reply:
@jeffrosenberg Well, it's a little bit more complicated than that. I had the `<TargetPath>` set here because otherwise I wouldn't get a `.dacpac` as the output, but a `.dll` even though I had set `<TargetExt>`. I now realized that we needed to have the `<TargetExt>` property set in the `Sdk.props` file rather than `Sdk.targets`. The props file is imported first, so setting the `<TargetExt>` there makes sure that by the time the `Sdk.targets` file is imported the `<TargetPath>` property is set to the correct value automatically.